test/library/namespace.go

```diff
@@ -0,0 +1,59 @@
+package library
```

Comment:
good call on making this exportable package to be used by e2e and e2e-preferred-host. We can move client creation logic here.